lib/queue_classic_plus/update_metrics.rb

```diff
@@ -18,7 +18,7 @@ def self.metrics
       {
         jobs_queued: jobs_queued,
         jobs_scheduled: jobs_scheduled,
-        max_created_at: max_age("created_at"),
+        max_created_at: max_age("created_at", "created_at = scheduled_at"),
```

Comment:
Why not do `scheduled_at > now()`? Whenever I do date1 == date2, I worry about millisecond difference.